baselines/commoncontrols.md

```diff
@@ -985,14 +985,13 @@ Required system-defined alerting rules, as listed in the Policy group descriptio
 4.  From the drop-down menu, select **Type**.
 5.  Select the **System defined** check box.
 6.  Click **Apply**.
-7.  A list of system defined rules displays. Select one of the rules from the list by clicking the table row for that rule—for example, the Device compromised rule.
-8.  From the Rule details page, you can view the conditions and actions for the rule—for example, to confirm if email notifications are turned on, and to confirm the recipients for those email notifications.
-9. Click **Edit Rule**.
-10. Click **Next: View Conditions**.
-11. Click **Next: Add Actions**.
-12. From the Actions page, you can change the severity for the alert to High, Medium, or Low, send an alert to the alert center if the rule's conditions are met, set up admin email notifications, and specify recipients for those notifications.
-13. Click **Next: Review**.
-14. Review the updated rule details, and then click **Update Rule**.
+7.  A list of system defined rules displays. For each of the rules from listed above, edit the configuration:
```

Comment:
Looks good! One more thing, sorry I didn't notice this the first time, there's a small typo:
```suggestion
7.  A list of system defined rules displays. For each of the rules listed above, edit the configuration:
```